framework/auth.js

```diff
@@ -232,7 +232,18 @@ function getAndroidBiometricAlert(cancelCallback) {
 	inner.add(fingerprintView);
 	inner.add(fingerprintLabel);
 	wrapper.add(inner);
-	var dialog = Dialog.confirm("Touch ID", L("auth_biometric_reason"), [
+
+	var dialogTitle = L("auth_biometric_fingerprint_title");
+	var dialogDescription = L("auth_biometric_fingerprint_reason");
+
+	if (_.isEmpty(dialogTitle)) {
+		dialogTitle = L("auth_biometric_title");
+	}
+	if (_.isEmpty(dialogDescription)) {
```

Comment:
Why soo verbose?

Isn't better?

```
L("auth_biometric_fingerprint_title") || L("auth_biometric_title")
```

Reply:
I added `Util.getFirstLocalizedString()` to improve this part.